docker-compose.yml

```diff
@@ -183,6 +186,14 @@ services:
       resources:
         ipv4_address: 172.20.0.100
 
+  test.httpbin.docker.local:
```

Comment:
@ReactorScram won't it already be available as `test.httpbin.docker.local` if you define it as `test.httpbin`? Otherwise, it will be available at `test.httpbin.docker.local.docker.local`

Reply:
Yep thanks, I'll fix that in the main PR